### ISSUE TITLE
Not able to start electron app

### ISSUE THREAD

**Issue by stripathix** [capacitor-community/electron#93]
**Describe the bug**
When i try to start an electron app then it fails with a weird log. Electron platform does get added successfully but on starting electron app it crash
```
shashwattripathi@Shashwats-MacBook-Pro electron-test % npx cap add @capacitor-community/electron

> @capacitor-community/electron@1.4.0 capacitor:add /Users/shashwattripathi/projects/electron-test/node_modules/@capacitor-community/electron
> node dist/cap-scripts.js add

✔ Adding Electron platform in 25.98s
✔ Updating Electron plugins in 12.87ms

shashwattripathi@Shashwats-MacBook-Pro electron-test % npx cap open @capacitor-community/electron

> @capacitor-community/electron@1.4.0 capacitor:open /Users/shashwattripathi/projects/electron-test/node_modules/@capacitor-community/electron
> node dist/cap-scripts.js open

⠴ Opening Electron platformError: undefined
✖ Opening Electron platform: 
(node:17308) UnhandledPromiseRejectionWarning: 
> electron-test@1.0.0 electron:start /Users/shashwattripathi/projects/electron-test/electron
> npm run build && electron ./


> electron-test@1.0.0 build /Users/shashwattripathi/projects/electron-test/electron
> tsc

node_modules/@types/fs-extra/index.d.ts(232,38): error TS2694: Namespace '"fs"' has no exported member 'BaseEncodingOptions'.
node_modules/@types/fs-extra/index.d.ts(233,38): error TS2694: Namespace '"fs"' has no exported member 'BufferEncodingOption'.
node_modules/@types/fs-extra/index.d.ts(234,38): error TS2694: Namespace '"fs"' has no exported member 'BaseEncodingOptions'.
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! electron-test@1.0.0 build: `tsc`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the electron-test@1.0.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/shashwattripathi/.npm/_logs/2021-03-21T08_29_09_297Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! electron-test@1.0.0 electron:start: `npm run build && electron ./`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the electron-test@1.0.0 electron:start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/shashwattripathi/.npm/_logs/2021-03-21T08_29_09_332Z-debug.log

(Use `node --trace-warnings ...` to show where the warning was created)
(node:17308) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:17308) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

shashwattripathi@Shashwats-MacBook-Pro electron-test % 

```

**To Reproduce**

Here is a project that i added using Ionic CLI 
https://github.com/stripathix/electron-test
Commands sequence i used in process
```
ionic start
// Then used vue project with tabs to setup project
npm run build
npm i @capacitor-community/electron
npx cap add @capacitor-community/electron
npx cap open @capacitor-community/electron.  ->>>> here it fails
```

**Expected behavior**
On using npx cap add electron application works as electron app but in log it says it is depricated and asks to use capacitor-communit/electron

**Screenshots**
<img width="1280" alt="Screenshot 2021-03-21 at 2 07 45 PM" src="https://user-images.githubusercontent.com/8480351/111898888-d1fc7f00-8a4e-11eb-8d2a-085bf7db0976.png">

**Desktop (please complete the following information):**

- OS:  MacOS BigSur 11.2.3
- Browser Chrome

**Comment by IT-MikeS** [comment]
Please try to cd into the electron folder and run `npm i @types/fs-extra@9.0.7`

It's a known issue with the type defs for fs-extra 9.0.8

**Comment by obrunopolo** [comment]
I also had to update other packages, since I could not start app after migrating from original electron platform. I followed @IT-MikeS suggestion (but using --save-dev) and also had to update `electron-is-dev` to latest (inside electron folder).

**Comment by IT-MikeS** [comment]
This should be corrected now as of v1.4.1
